_Classes/DeusEx/Classes/DeusExPlayer.uc

```diff
@@ -557,8 +557,6 @@ const DRUG_CRACK = 2;
 
 var bool autosave;                                                              //Sarge: Autosave tells the Quicksave function to make an autosave instead
 
-var travel DeusExWeapon lastMeleeWeapon;                                           //Sarge: Stores our last melee weapon, for use when left-click frobbing crates and other breakables
```

Comment:
In addition to this, the `bIsMeleeWeapon` variable in `DeusExWeapon` should to be removed as well, since it was telling the system which weapons should be added as last melee weapons. Now that this system isn't used anymore, that variable should go too